build/tutorials/tutorials-contest/avalanche-erc721-tutorial/utils/tokens.js

```diff
@@ -0,0 +1,5 @@
+const convertTokensToWei = (n) => {

```

Comment:
Add return values for all functions

Reply:
Thanks for the input! Everything else is clear, but I'm not sure what you mean here exactly. I am returning a value in this function?